sqlobject/src/main/java/org/jdbi/v3/sqlobject/mixins/GetHandle.java

```diff
@@ -41,7 +41,14 @@
      * @throws CallbackFailedException Will be thrown if callback raises an exception. This exception will
      *                                 wrap the exception thrown by the callback.
      */
-    <ReturnType> ReturnType withHandle(HandleCallback<ReturnType> callback) throws CallbackFailedException;
+    default <ReturnType> ReturnType withHandle(HandleCallback<ReturnType> callback) throws CallbackFailedException {
+        try {
+            return callback.withHandle(getHandle());
+        }
+        catch (Exception e) {
+            throw new CallbackFailedException(e);
```

Comment:
This is already such a big pull request, and the cognitive load was getting too high already. That's why I opened the separate PR for the exception transparency stuff.

This change was just about migrating from `GetHandleHandler` to a default method.